src/routes/_utils/blurhash.js

```diff
@@ -0,0 +1,49 @@
+import BlurhashWorker from 'worker-loader!../_workers/blurhash' // eslint-disable-line
```

Comment:
I couldn't get `worker-loader` working when added to `client.config.js` for some reason. Standard complains about the inline use of the loader, disabled it for now.

Reply:
That's fine; I don't really care whether the `worker-loader` is referenced in the Webpack config or in the import.